src/cli/cmd/version/version.go

```diff
@@ -24,7 +24,7 @@ func NewCmd() *cobra.Command {
 }
 
 func cmdRunE(cmd *cobra.Command, args []string) (err error) {
-	cmd.Print(format(build.Version, build.Date))
+	fmt.Fprintf(cmd.OutOrStdout(), format(build.Version, build.Date))
```

Comment:
@cc-shekher I might have done that sorry! But maybe I'm miss-understanding something, I thought `cmd.Print` was short-hand for `fmt.Fprintf(cmd.OutOrStdout(),`. not true?

Reply:
@kanaksinghal If you see the definition of `cmd.Print` it is as below:

`func (c *Command) Print(i ...interface{}) {
	fmt.Fprint(c.OutOrStderr(), i...)
}`

using `cmd.OutOrStdout()` will explicitly override that.